tests/document/instance.py

```diff
@@ -954,11 +954,13 @@ def UUID(i):
         self.assertEqual(w1.save_id, UUID(1))
         self.assertEqual(w1.count, 0)
 
-        # mismatch in save_condition prevents save
+        # mismatch in save_condition prevents save and raise exception
         flip(w1)
         self.assertTrue(w1.toggle)
         self.assertEqual(w1.count, 1)
-        w1.save(save_condition={'save_id': UUID(42)})
+        self.assertRaisesRegexp(OperationError,
```

Comment:
Just replace it with assertRaises() we do not loose that much of test quality, it's just about the message